boards/arm/nrf52840_papyr/Kconfig.defconfig

```diff
@@ -0,0 +1,29 @@
+# Kconfig - Electronut Labs Papyr board configuration
+#
+# Copyright (c) 2018 Electronut Labs
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52840_PAPYR
+
+config BOARD
+	default "nrf52840_papyr"
+
```

Comment:
What about `I2C_0` and `SPI_1`? Wouldn't it be good to also enable these instances by default here?